### ISSUE TITLE
[Other Issues]: Can there be a 2109.1 ASAP as the Messagebox bug in 2108.## is really annoying

### ISSUE THREAD

**Issue by Smurf-IV** [Krypton-Suite/Standard-Toolkit#330]

**Comment by Wagnerp** [comment]
Not until next week as I'm away until Sunday

**Comment by Wagnerp** [comment]
What is the bug about?

**Comment by Smurf-IV** [comment]
> 
> 
> What is the bug about?

#242

**Comment by Wagnerp** [comment]
Wasn't this fixed in 2109?

**Comment by Smurf-IV** [comment]
Yes, But 2109 is not useable due to button images being removed and the other fixes that have had to be done.

**Comment by Wagnerp** [comment]
> Yes, But 2109 is not useable due to button images being removed and the other fixes that have had to be done.

So the buttons aren't usable?

**Comment by Smurf-IV** [comment]
Not for use in a QA environment..
It would fail every test,
Along with the other fixes for actual development of menu etc.
I've had to go back to 2108.1 for Code testing.

**Comment by Wagnerp** [comment]
> Not for use in a QA environment..
> It would fail every test,
> Along with the other fixes for actual development of menu etc.
> I've had to go back to 2108.1 for Code testing.

Sounds like the 'Click' events are not firing

**Comment by Smurf-IV** [comment]
No it was the removal of the image bug, and the designer bug (And a few others) that are stopping the use of 2109.

**Comment by Wagnerp** [comment]
So it's related to the image disappearing bug?

**Comment by Smurf-IV** [comment]
> 
> 
> So it's related to the image disappearing bug?

What.. No..
Start Again.. 
2109 is not suitable for QA testing because of a number of bugs. (_Some_ listed above)
2108.# fails QA because of the MessageBox clipping (Hence the title)

**Comment by Wagnerp** [comment]
Ah I now understand, sorry I'm away from a PC, so I can't replicate or test the bug until Monday

**Comment by Wagnerp** [comment]
@Smurf-IV Solved the image click bug. Can this be closed or not?

**Comment by Smurf-IV** [comment]
No, Can you merge _ALL_ the other changes into canary 2109.1 since 2109 went out, as well please ?

**Comment by Wagnerp** [comment]
> No, Can you merge _ALL_ the other changes into canary 2109.1 since 2109 went out, as well please ?

@Smurf-IV Which fixes need to go into 2109.1 from 2110, as a merge from alpha to canary won't work, unless you delete the entire source folder inside of canary? Or another option would be to pull 2109, and wait until 2110?

**Comment by Smurf-IV** [comment]
- Restore layout to orginal Krypton Intent
#338 by Smurf-IV was merged 16 hours ago • Approved
- Make sure values can accept a 16*16 icon
#337 by Smurf-IV was merged 17 hours ago

Alpha 178 designers
#334 by Smurf-IV was merged yesterday • Approved

* Fix #323
#328 by Wagnerp was merged 8 days ago

- And the Changelog
#326 by Smurf-IV was merged 9 days ago • Approved
- Sort out the Callback actions that were removed
#325 by Smurf-IV was merged 9 days ago • Approved

* Alterations
#322 by Wagnerp was merged 10 days ago

**Comment by Wagnerp** [comment]
Published - 13/09/2021

**Comment by Smurf-IV** [comment]
@Wagnerp 
Something has been mismerged:

![image](https://user-images.githubusercontent.com/2418812/133062423-0725780a-6d75-45e8-8fd3-2d7db37aa0b8.png)

**Comment by Wagnerp** [comment]
> @Wagnerp
> Something has been mismerged:
> 
> ![image](https://user-images.githubusercontent.com/2418812/133062423-0725780a-6d75-45e8-8fd3-2d7db37aa0b8.png)

Was the TFMs meant to be cut down in 2110?

**Comment by Smurf-IV** [comment]
If that is when V6 is being released, then yes.

**Comment by Smurf-IV** [comment]
I mean, "The next" version as in November will have the TFM's "trimmed", so these Canary builds should be reflecting that as part of their testing.

**Comment by Wagnerp** [comment]
According to the changelog, the change happened in build 2109

![image](https://user-images.githubusercontent.com/949607/133064853-291a96eb-81f2-43a0-a73e-61ae115593f8.png)

Is there a bug in `MissingFrameworkAPIs`?

**Comment by Smurf-IV** [comment]
`MissingFrameworkAPIs` was deleted in alpha, so why did the build succeed for 2109.1 ? 
and yet it still appears to be referenced in the ribbon dll ?

Was that not merged into 2109 ?

https://github.com/Krypton-Suite/Standard-Toolkit/commit/79fb4e2135ba42349e7e1d23ab69dae612d1bb3e

![image](https://user-images.githubusercontent.com/2418812/133068529-c0e8583d-11d3-43f5-bf0b-f3e51329ea96.png)

**Comment by Wagnerp** [comment]
Did something not get changed in alpha-enh#290?

**Comment by Smurf-IV** [comment]
Have you withdraw 2109 and 2109.1 ? I cannot test with the Ribbon Examples 2019 Nuget solution ?

**Comment by Wagnerp** [comment]
I'm going to pull them, as there seems to be issues

**Comment by Smurf-IV** [comment]
Ok.. This is what I have done..
1 Cleaned Ribbon Examples 2019 Dev solution
2 close solution
3 Got out alpha
4 Open Ribbon Examples 2019 Dev solution
5 Build run Keytips + .. test project

and then repeated with Canary branch and Canary 2109.1

So the only things I can conclude (Without having the 2109.1 nugets available) is that either 
- the Ribbon 2109.1 was pulling 2108.# of the toolkit for some reason 
- or 2108.1 Ribbon was not replaced by 2109.1 in either Nuget or your push of 2109.1 ..

**Comment by Wagnerp** [comment]
Build 2109 & 2109.1 has been revoked due to bugs as discussed. Closing thread.
